aldryn_people/models.py

```diff
@@ -203,14 +208,17 @@ def get_absolute_url(self, language=None):
 
     def get_vcard(self, request=None):
         if self.primary_group:
-            group_name = self.primary_group.safe_translation_getter('name')
+            group_name = self.primary_group.safe_translation_getter(
+                'name', default="Group: {0}".format(self.primary_group.pk))
```

Comment:
this check is done twice, the group_name variable is not used until the second check so this group_name block can be moved down to the second check